dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/controllers/FileOpenController.java

```diff
@@ -39,6 +40,7 @@
 	private static final int WIDTH = 550;
 	private static final int ANIM_DURATION = 250;
 
+	@SuppressWarnings("unused") @FXML private AnchorPane container;
 	@SuppressWarnings("unused") @FXML private GridPane fileOpenPane;
```

Comment:
All `fileOpenPane` uses should be replaced with `container`. When that's done, `fileOpenPane` can be removed.